depscan/lib/csaf.py

```diff
@@ -0,0 +1,1663 @@
+import logging
+import json
+import os
+import re
+from copy import deepcopy
+from datetime import datetime
+from json import JSONDecodeError
+
+from requests import HTTPError
+
+import requests
+import toml
+import tomli
+
+from depscan.lib.logger import LOG
+
+
+CWE_MAP = {
```

Comment:
Wondering if this can be loaded from this csv file. 

https://cwe.mitre.org/data/csv/699.csv.zip

Reply:
@prabhu It could be, but that list appears to be incomplete - and I would still have to create this dictionary for air-gapped environments to avoid downloading the file. 

Unless you know how this sort of file could be included in a python package? I couldn't figure a way when I was trying to do it for the csaf.toml.

Maybe the missing entries don't matter for some reason, but I'm not aware if that is the case or not.